docker/start_amaru.sh

```diff
@@ -26,10 +26,10 @@ fi
 
 # keep stack traces for troubleshooting purposes
 export RUST_BACKTRACE=full
-export AMARU_LOG=debug
-export AMARU_TRACE=gasket=error,debug
 
-exec cargo run --profile dev -- --with-json-traces daemon \
+export AMARU_LOG=gasket=error,amaru=debug,amaru::stages::consensus::forward_chain=info,info
```

Comment:
Isn't explicitly setting `amaru::stages::consensus::forward_chain` redundant?

Reply:
I don't think so, I tried locally and they were still very chatty.